launch/launch/event_handler.py

```diff
@@ -36,6 +36,9 @@ class EventHandler:
     via the context's locals, e.g. `context.locals.event`
     As another example, getting the name of the event as a Substitution:
     `launch.substitutions.LocalSubstitution('event.name')`.
+
+    Child classes should implement the _handle(event: Event, context: 'LaunchContext')
+    function instead of overriding the handle() function.
```

Comment:
The easiest thing to do would have been for child classes to override `self.entities` instead, but most of them also needed access to the context, and I didn't want to break the API. Alternative ideas are welcome.

Reply:
I'm fine with not changing entities, but see my review for a subtle change to the approach documented here.